### PR TITLE
fix: toggle flag aliases

### PR DESCRIPTION
**Pull request by dbolson**
Adds the `flags toggle-on/toggle-off` actions back.

Running `ldcli flags --help` shows the new actions:
```
Available Commands:
  copy                           Copy feature flag
  create                         Create a feature flag
  create-migration-safety-issues Get migration safety issues
  delete                         Delete feature flag
  get                            Get feature flag
  get-status                     Get feature flag status
  get-status-across-environments Get flag status across environments
  list                           List feature flags
  list-expiring-context-targets  Get expiring context targets for feature flag
  list-expiring-user-targets     Get expiring user targets for feature flag
  list-statuses                  List feature flag statuses
  toggle-off                     Turn a feature flag off
  toggle-on                      Turn a feature flag on
  update                         Update feature flag
  update-expiring-targets        Update expiring context targets on feature flag
  update-expiring-user-targets   Update expiring user targets on feature flag
```

**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: cmd/resources/resources.go [hunk 1]
```diff
@@ -395,7 +395,8 @@ Global Flags:
 Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
   {{rpad .CommandPath .CommandPathPadding}} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableSubCommands}}
 
-Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}`
+Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
+`
```
Comment (dbolson) [inline cmd/resources/resources.go:399]: Added this newline so the prompt wouldn't end up on the last line as the help output, consistent with the other inline templates.

File: cmd/root.go [hunk 2]
```diff
@@ -182,6 +183,14 @@ func NewRootCommand(
 	cmd.AddCommand(resourcecmd.NewResourcesCmd())
 	resourcecmd.AddAllResourceCmds(cmd, clients.ResourcesClient, analyticsTrackerFn)
 
+	// add non-generated commands
```
Comment (dbolson) [inline cmd/root.go:186]: We can add other aliased or other commands here.

File: internal/resources/mock_client.go [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package resources
+
+import "net/url"
+
+type MockClient struct {
```
Comment (dbolson) [inline internal/resources/mock_client.go:5]: I ended up not going with the testify mock code. If you'd prefer that way, I can change this.

File: cmd/flags/toggle.go [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	"github.com/spf13/viper"
+
+	"ldcli/cmd/cliflags"
+	"ldcli/cmd/validators"
+	"ldcli/internal/errors"
+	"ldcli/internal/output"
+	"ldcli/internal/resources"
+)
+
+func NewToggleOnCmd(client resources.Client) *cobra.Command {
```
Comment (k3llymariee) [inline cmd/flags/toggle.go:16]: I think we also need to set the usage template for these, in order to parse out the required/optional flags

**Review by k3llymariee — APPROVED** [review]
just a comment about updating the usage template but that can be added in a follow-up

File: cmd/flags/toggle.go (hunk 4 above)
Comment (dbolson) [reply cmd/flags/toggle.go:17]: Updated. Help now looks like this
```
Turn a feature flag on

Usage:
  ldcli flags toggle-on [flags]

Required flags:
      --environment string   The environment key
      --flag string          The feature flag key
      --project string       The project key

Optional flags:


Global Flags:
  -h, --help                  Help for this command
      --access-token string   LaunchDarkly access token with write-level access
      --analytics-opt-out     Opt out of analytics tracking
      --base-uri string       LaunchDarkly base URI (default "https://app.launchdarkly.com")
  -o, --output string         Command response output format in either JSON or plain text (default "plaintext")
```

**Pull request merged**